bigtop-deploy/puppet/modules/hadoop/templates/core-site.xml

```diff
@@ -169,4 +169,11 @@
     <value>com.quantcast.qfs.hadoop.QuantcastFileSystem</value>
   </property>
 <% end -%>
+
+<% if @kms_host %>
+  <property>
+    <name>hadoop.security.key.provider.path</name>
+    <value>kms://http@<%= @kms_host %>:<%= @kms_port %>/kms</value>
```

Comment:
I'm zero knowledge about this, just wondering can this be https?

Reply:
Yes. We need additional configurations in files such as ssl-client.xml, ssl-server.xml and server.xml (of Tomcat) for that. I would like to address that in another JIRA since all services of HDFS and YARN should be cared when we enable HTTPS on Web-UI and REST API.

Reply:
Sure. No problem and thanks for the explanation.